notifier/texters.rst

```diff
@@ -14,6 +14,9 @@ you to send SMS messages::
 
     // src/Controller/SecurityController.php
     namespace App\Controller;
+    
```

Comment:
Feel free to also add the namespace for the Route-annotation, but as far as I am concerned those 2 are more important for this example